app/src/main/java/com/example/bookwormadventuresdeluxe2/FirebaseUserGetSet.java

```diff
@@ -179,6 +185,61 @@ public void onComplete(@NonNull Task<Void> task)
                 });
     }
 
+    /**
+     * Increment User notification count
+     */
+    public static void incrementNotificationCount(String userId)
+    {
+        FirebaseFirestore.getInstance().collection("Users")
+                .document(userId)
+                .update("notificationCount", FieldValue.increment(1));
+    }
+
+    /**
+     * Reset User notification Count and Delete Notifications
+     */
+    public static void resetNotifications(String userId)
+    {
+        /* Update UserCredentialAPI*/
+        UserCredentialAPI.getInstance().setNotificationCount(null);
+        /* set notification count to null */
+        firebase.collection("Users")
```

Comment:
Please try and avoid hardcoding the collection names

Reply:
My apologies. Was working on this at 4 AM lol. Fixed